controllers/validator.go

```diff
@@ -75,7 +76,12 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 	// validate non-admin enable and tech-preview-ack
 	if r.checkNonAdminEnabled() {
 		if !(dpa.Spec.UnsupportedOverrides[oadpv1alpha1.TechPreviewAck] == TrueVal) {
-			return false, errors.New("in order to enable/disable the non-admin feature please set dpa.spec.unsupportedOverrides[tech-preview-ack]: 'true'")
+			return false, errors.New("in order to enable the non-admin feature please set dpa.spec.unsupportedOverrides[tech-preview-ack]: 'true'")
+		}
+		if r.dpa.Spec.NonAdmin.EnforceBackupSpec != nil {
```

Comment:
added this to avoid headaches with users creating bad configuration

I think this is NAC responsibility, not user. If user can set this, then all NonAdminBackups will fail (or only of a specific namespace) 

do you agree with this safety check?

(if yes, other fields should be added here?)

Reply:
Hmm..I think maybe we should keep this optional and give a free reign to the admins.

Reply:
ok